Solutions/CiscoUmbrella/Hunting Queries/CiscoUmbrellaHighValuesOfUploadedData.yaml

```diff
@@ -13,6 +13,6 @@ query: |
   | where EventType == "proxylogs"
   | where TimeGenerated > ago(timeframe)
   | summarize sum(DstBytes) by SrcIpAddr,DstIpAddr
-  | sort by sum_DstBytes desc
+  | sort by sum_SrcBytes desc
```

Comment:
We sum by DstBytes above so this will fail. You either need to update the above Sum or sort by sum_DstBytes